docs/A-Introduction/compilers.mdx

```diff
@@ -23,7 +23,23 @@ We transform the program instructions and program data into the bits and bytes t
 
 Programming languages demand completeness and greater precision than human languages. The ultimate interpreter of any computer program is the hardware. Hardware cannot interpret intent or nuance, and we need to provide much more detail in our instructions than we do in casual conversations amongst ourselves. In this sense, programming is a more arduous task than writing a formal report that someone else will read.
 
-!["Human languages, Programming languages and machine languages" ](/img/languages.png)
+```mermaid
+%%{ init: { 'flowchart': { 'useMaxWidth': false } } }%%
+flowchart
+  subgraph title1[Human Language]
+      subgraph title2[Programming Language]
+        title3([Machine Language]):::title
+      end
+    end
+    title1:::header1
+    title2:::header2
+    title3:::header3
+    classDef header1 font-size: 12px, font-weight: 600;
+    classDef header2 font-size: 10px, font-weight: 600, fill:lightblue;
+    classDef header3 font-size: 9px, font-weight: 600, fill:orange;
+```
+
+<!-- !["Human languages, Programming languages and machine languages" ](/img/languages.png) -->
```

Comment:
```suggestion
```

I know this is nit-picking, but maybe it's a better idea to remove the old code instead of commenting it out?

Reply:
@devils2ndself  Thanks for the feedback. Yes, I would have removed if it was certain that my mermaid diagrams replaces the pictures. I thought it's good to leave them so that reviewer can compare both. I will push another commit without these comments.